docs/web_advanced.rst

```diff
@@ -190,21 +205,23 @@ But for development it's very convenient to handle static files by
 aiohttp server itself.
 
 To do it just register a new static route by
-:meth:`UrlDispatcher.add_static` call::
+:meth:`RouteTableDef.static` or :func:`static` calls::
+
+   app.add_routes([web.static('/prefix', path_to_static_folder)])
 
-   app.router.add_static('/prefix', path_to_static_folder)
+   routes.static('/prefix', path_to_static_folder)
```

Comment:
Is `routes.static` correct here? The examples below all use `static`

Reply:
They are alternatives as pointed a couple lines above: `RouteTableDef.static()` and `web.static()` coexists.

But I should use `web.static` below to emphasize `web.static()` version usage